docs/learning/training.md

```diff
@@ -0,0 +1,137 @@
+Non-coding project participants should also become familiar with GitHub's user interface,
+and have a understanding of how to write issues and use Markdown.
+
+### 1. Set up a GitHub account
+
+If you already have an account, great!
+If not, go ahead and set one up by going to [GitHub's pricing page](https://github.com/pricing)
+and selecting the option for a free account. Then:
+
+1. Enter a username
+1. Enter your email (we suggest work email, you can associate other emails later if you like)
+1. Enter a password
+1. Verify that you're human by solving a puzzle
+1. Click create account
+1. An email will be sent to you for verification. Verify your email following those instructions
+
+You'll use this account or one you already have to participate in the modern data stack accelerator.
+
+### 3. Read GitHub's [guide to git](https://github.com/git-guides/)
+
+This is a good introduction to the mechanics of git (separate from GitHub as a code hosting platform).
+A large amount of development happens locally on your machine,
+and being fluent with concepts like cloning, pushing, pulling, and committing
+will make everything that follows much easier.
+
+### 3. Go through the [introduction to GitHub tutorial](https://github.com/skills/introduction-to-github).
+
+This should take about an hour, and should be taken by all data engineers and data analysts on the project.
+You will learn how to create a branch, commit a change, open a pull request, and merge your change.
+
+### 4. Learn how to [author GitHub-flavored Markdown](https://github.com/skills/communicate-using-markdown)
+
+This should take about an hour, and involves no code.
+It is useful for many project members to learn this material,
+including those who are not doing day-to-day coding,
+as it is used in issue tracking, cross referencing, and other project-management tooling.
+
+### 5. Learn how to [review pull requests on GitHub](https://github.com/skills/review-pull-requests)
+
+This should take about an hour, and should be taken by all data engineers and data analysts on the project.
+You will learn how to open pull requests, review them, suggest changes, and merge them.
+Any change to the project codebase should go through this process,
+as it allows for peer review, automated checks and tests (CI), and automated deployments (CD).
+
+## Snowflake
+
+**Goals of training**
+
+* Understand what Snowflake is, and how cloud data warehouses differ from traditional on-prem OLTP databases.
+* Become familiar with Snowflake terminology (warehouse, database, role etc).
+* Have a basic understanding of how to use the Snowflake user interface.
+
+**Who should go through this?**
+Data engineers should go through this training,
+
+### 1. Take Snowflake's [Data Warehousing Workshop](https://learn.snowflake.com/en/courses/uni-essdww101/)
+
+This covers the fundamentals of the Snowflake Data Warehouse.
+It should take 6-8 hours.
```

Comment:
@britt-allen I don't have much information about the content here, do you have anything you want to add about this workshop?

Reply:
Yes, I can copy content from here: https://docs.google.com/document/d/1fCS0_8WANH1YZF53Jk006e0UHoygnb_sJvUF4olF-jc/edit